mxcube3/ui/reducers/sampleview.js

```diff
@@ -7,7 +7,7 @@ const initialState = {
   width: 0,
   height: 0,
   lightOn: false,
-  motors: { Light: {}, Phi: {}, PhiY: {}, PhiZ: {}, Sampx: {}, Sampy: {}, Zoom: {} },
+  motors: {},
```

Comment:
Agreed, but then it would be better to display sample view, then having an error notification than simply don't displaying it at all.